gnomad/resources/grch38/gnomad.py

```diff
@@ -406,3 +418,67 @@ def release_vcf_path(data_type: str, version: str, contig: str) -> str:
     contig = f".{contig}" if contig else ""
     version_prefix = "r" if version.startswith("3.0") else "v"
     return f"gs://gcp-public-data--gnomad/release/{version}/vcf/{data_type}/gnomad.{data_type}.{version_prefix}{version}.sites{contig}.vcf.bgz"
+
+
+def gnomad_gks(
+    version: str,
+    variant: str,
+    data_type: str = "genomes",
+    by_ancestry_group: bool = False,
+    by_sex: bool = False,
+    vrs_only: bool = False,
+    custom_path: str = None,
+) -> dict:
+    """
+    Call get_gks() and return VRS information and frequency information for the specified gnomAD release version and variant.
```

Comment:
here and below, indent text to keep in line

Reply:
I'm a bit confused - this looks like the rest of the code in annotation.py and isn't returning any whitespace errors and isn't changed by 'black' formatting. is there something I'm missing on this ?

Reply:
nm, looks fine now